cli/flox/doc/flox-update.md

```diff
@@ -0,0 +1,56 @@
+---
+title: FLOX-UPDATE
+section: 1
+header: "flox User Manuals"
+...
+
+
+# NAME
+
+flox-update - update the global base catalog or an environment's base catalog
+
+# SYNOPSIS
+
+```
+flox [ <general-options> ] update
+     [--global | (-d=<path> | -r=<owner/name>)]
+```
+
+# DESCRIPTION
+
+Update the global base catalog if `--global` is specified,
+or update an environment's base catalog.
```

Comment:
```suggestion
Update an environment's base catalog,
or the global base catalog if `--global` is specified,
```

Reply:
I've put `--global` before environment throughout (NAME, SYNOPSIS, later on in DESCRIPTION, OPTIONS), so I think it's more consistent to leave this as is

Reply:
In general I think it's better to describe the basic functionality first, and then mention how it behaves with options. I also don't think this _has_ to be consistent with the order in which options are listed.

Reply:
Fixing